tests/AiDotNet.Tensors.Benchmarks/BaselineRunners/py/run_codegen_bakeoff.py

```
@@ -27,6 +28,18 @@
 RUNS = 3
 
 
+def conv_transpose_contract():
+    """Returns N,C,IH,IW,OH,OW,output_padding from the .NET catalog authority."""
+    raw = os.environ.get("BAKEOFF_CONV_TRANSPOSE_CONTRACT", "")
+    try:
+        values = tuple(int(value) for value in raw.split(","))
+    except ValueError as exc:
+        raise RuntimeError("invalid BAKEOFF_CONV_TRANSPOSE_CONTRACT") from exc
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)